tests/trainer/logging_/test_distributed_logging.py

```diff
@@ -66,3 +67,37 @@ def test_global_zero_only_logging_ddp_spawn(tmpdir):
         weights_summary=None,
     )
     trainer.fit(model)
+
+
+class LoggerCallsObserver(Callback):
+
+    def on_before_accelerator_backend_setup(self, trainer, pl_module):
+        assert not trainer.logger.method_calls
+        assert not os.listdir(trainer.logger.save_dir)
+
+    def on_train_start(self, trainer, pl_module):
+        assert trainer.logger.method_call
+        trainer.logger.log_hyperparams.assert_called_once()
+        trainer.logger.log_graph.assert_called_once()
```

Comment:
nit: can you put this class inside the test? makes it easier to track that is only used in one place

Reply:
moved :)